pytential/symbolic/dof_desc.py

```diff
@@ -275,6 +275,8 @@ def as_dofdesc(desc: DOFDescriptorLike) -> DOFDescriptor:
 
 # {{{ type annotations
 
+DEFAULT_DOFDESC = DOFDescriptor()
```

Comment:
Renamed this guy `DEFAULT_DOFDESC`, hope that's OK.

Reply:
Yeah, definitely better! Not sure why I named it empty..